ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/config/DataEdgeConfig.java

```diff
@@ -35,4 +69,134 @@ public TaskExecutor getAsyncExecutor() {
         executor.initialize();
         return executor;
     }
+
+    @Bean
+    public Ga4ghService initGa4ghServiceImpl(final RestTemplate restTemplate,
+                                             @Value("${ega.aai.proxy.url}") final String egaAAIProxyURL,
+                                             @Value("${ega.aai.proxy.basic-authorization}") final String egaAAIProxyBasicAuth) throws URISyntaxException, MalformedURLException {
+        return new Ga4ghServiceImpl(
+                restTemplate,
+                new URL(egaAAIProxyURL),
+                egaAAIProxyBasicAuth
+        );
+    }
```

Comment:
Could you please format this part of the code? :-) Nothing wrong with the code itself, it's only the format is a bit difficult to read.

Reply:
As discussed, formatting not needed.